tests/cypress/integration/canvas3d_functionality/case_64_canvas3d_functionality_cuboid.js

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName } from '../../support/const_canvas3d';
+
+context('Canvas 3D functionality. Add cuboid.', () => {
+    const caseId = '64';
+
+    const screenshotsPath = 'cypress/screenshots/canvas3d_functionality/case_64_canvas3d_functionality_cuboid.js';
+
+    before(() => {
+        cy.openTaskJob(taskName);
+        // Prepare screenshots to compare
+        cy.get('.cvat-canvas3d-perspective').screenshot('canvas3d_perspective_before_all');
+        ['topview', 'sideview', 'frontview'].forEach((view) => {
+            cy.get(`.cvat-canvas3d-${view}`).find('.cvat-canvas3d-fullsize').screenshot(`canvas3d_${view}_before_all`);
+        });
+    });
+
+    describe(`Testing case "${caseId}"`, () => {
+        it('Add cuboid.', () => {
+            cy.get('.cvat-draw-cuboid-control').trigger('mouseover');
+            cy.get('.cvat-draw-shape-popover').find('button').click();
+            cy.get('.cvat-canvas3d-perspective').dblclick();
+            cy.get('.cvat-canvas3d-perspective').screenshot('canvas3d_perspective_after_add_cuboid'); // The cuboid displayed
+            cy.compareImagesAndCheckResult(
+                `${screenshotsPath}/canvas3d_perspective_before_all.png`,
+                `${screenshotsPath}/canvas3d_perspective_after_add_cuboid.png`,
+            );
+        });
+
+        it('Cuboid interaction by mouse.', () => {
+            // The cuboid does not have time to change color so we make the mouse movement 3 times.
+            cy.get('.cvat-canvas3d-perspective')
+                .trigger('mousemove', 500, 200)
+                .trigger('mousemove', 400, 200)
+                .trigger('mousemove', 300, 200); // The cuboid should change a color after movement cursor from it
```

Comment:
Just a question: why exactly three movement is used here and further, not two or simple delay? Is it not working?

Reply:
@ActiveChooN 

Delays lead to indeterminate (when sometimes tests passed, sometimes tests failed). It is better to try escaping them if possible.

Reply:
Yes, this is done to exclude the use of wait(). The color of the cuboid does not change immediately when the mouse interacts with it.